apps/microtvm/zephyr/template_project/src/mlperftiny/submitter_implemented.cc

```diff
@@ -19,44 +19,189 @@
 
 #include "api/submitter_implemented.h"
 
+#include <string.h>
+#include <tvm/runtime/crt/logging.h>
 #include <tvm/runtime/crt/platform.h>
 #include <unistd.h>
 #include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/uart.h>
 #include <zephyr/kernel.h>
+#include <zephyr/sys/ring_buffer.h>
 
 #include "api/internally_implemented.h"
-#include "tvmruntime.h"
-#include "zephyr_uart.h"
+#include "crt_config.h"
+#include "tvm/output_data.h"
+#include "tvmgen_default.h"
+
+// ###############################################################
+// Model
+// ###############################################################
+#define MODEL_KWS 1
+#define MODEL_VWW 2
+#define MODEL_AD 3
+#define MODEL_IC 4
 
 static void* g_input_data;
-#if TARGET_MODEL == 3  // AD
+// TODO: use name instead of number directly
```

Comment:
nit: WDYM here? `#if TARGET_MODEL == MODEL_AD` seems fine.

Reply:
forgot to remove that comment. Removed now.